tests/common/timing_dump_vivado.tcl

```diff
@@ -0,0 +1,12 @@
+set fp [open [lindex $argv 1] w]
+read_checkpoint [lindex $argv 0]
+link_design
+foreach net [get_nets] {
+	if {[get_nets $net] != ""} {
+		if {[get_net_delays -of_objects [get_nets $net]] != ""} {
+			puts $fp "$net [get_property SLOW_MAX [lindex [get_net_delays -of_objects [get_nets $net]] 0]]"
```

Comment:
this will only work for the first branch of a net - and it's probably worth checking that Vivado and interchange both have the same idea as to what the first branch of a net is, unless we only consider nets with one sink (but that then means we don't test fanout dependence of the timing model)

Reply:
I've changed vivado script to dump all timing for nets, not only first sink.
Also STA dumps all sinks in `--compact` mode

Reply:
Looks good